--- gnuastro-0.19/lib/Makefile.am.orig	2023-01-19 21:53:09.471630170 +0100
+++ gnuastro-0.19/lib/Makefile.am	2023-01-19 21:56:16.093952483 +0100
@@ -69,8 +69,8 @@ AM_CPPFLAGS = -I\$(top_builddir)/bootstr
 lib_LTLIBRARIES = libgnuastro.la $(MAYBE_GNUMAKE)
 
 # Linking flags for the Gnuastro library.
-libgnuastro_la_LIBADD = $(top_builddir)/bootstrapped/lib/libgnu.la
-libgnuastro_la_LDFLAGS = -version-info $(GAL_LT_VERSION) $(CONFIG_LDADD) \
+libgnuastro_la_LIBADD = $(top_builddir)/bootstrapped/lib/libgnu.la $(CONFIG_LDADD)
+libgnuastro_la_LDFLAGS = -version-info $(GAL_LT_VERSION) \
                          -lc -no-undefined
 
 # Gnuastro's GNU Make extensions
