--- gnuastro-0.7/configure.ac.orig	2018-08-08 12:36:45.000000000 +0200
+++ gnuastro-0.7/configure.ac	2018-12-20 05:45:37.682824046 +0100
@@ -401,7 +401,7 @@
         outname=libtool_shell_test
         cprog=libtool_shell_test.c
         echo "#include <stdio.h>"                                 > $cprog
-        echo "int main(void){printf(\"success\\n\"); return 0;}" >> $cprog
+        printf 'int main(void){printf("success\\n"); return 0;}\n' >> $cprog
         ltargs="--quiet --tag=CC --mode=link $CC $cprog -O3 -o $outname"
         AS_IF( sh -c "$gnulibtool_exec $ltargs" > /dev/null 2>&1,
                [libtool_shell="sh"],
